### ISSUE TITLE
4.0.1.4 and 4.0.1.5 "Drag executable here to open" not working

### ISSUE THREAD

**Issue by exKitsune** [xmoeproject/KrkrExtract#65]
Trying to drag an exe onto the KrkrExtract window does nothing

**Comment by Veshurik** [comment]
Yeah, for me too (maybe because I'm trying it for Steam version of game?)
That game - https://vndb.org/v21676

**Comment by Veshurik** [comment]
No, it also for any other game. Nah.

**Comment by Cerzs** [comment]
Nope, it does not work with any game.

**Comment by aikoofujimotoo** [comment]
**Before doing this, back up the game's executable first**
Try using morkt's [setdll](https://github.com/morkt/GARbro/files/3989916/setdll.zip). Unpack them to the game's directory along with `KrkrExtract.dll` and run the following commands:

```bash
setdll /d:KrkrExtract.dll <program's exe>
```
After it, it should open the dialogue and do your stuff there.

Annoying but it works.
